### PR TITLE
Adding the stand-alone map component

### PR DESCRIPTION
**Pull request by shoshi**
screen shot 2018-10-29 at 3 52 35 pm

### CODE REVIEW

File: template/config.json [hunk 1]
```diff
@@ -9,5 +9,6 @@
   "googleAnalyticsId": "{{ googleAnalyticsId }}",
   "actionNetworkPetitionId": "{{ actionNetworkPetitionId }}",
   "donateUrl": "{{ donateUrl }}",
+  "mapboxToken": "pk.eyJ1Ijoia2VuZmZ0ZiIsImEiOiJjamlld2tzaTQwcXV6M3BxeXdlNXRnMXh1In0.A0ZjZjYMkyK7IvL9JLhUpg",
```
Comment (kenmickles) [inline template/config.json]: I know we've been re-using this token everywhere, but I think we should start creating new access tokens for each site.

**Review by kenmickles — COMMENTED** [review]
I think we should make the mapboxToken a config var. Otherwise, looks good.

File: template/config.json (hunk 1 above)
Comment (shoshi) [reply template/config.json]: Sounds good. Should this be a configurable key on site generation?

File: template/components/Map.vue [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+      },
+      set(newVal) {
+        this.$store.commit('setMapCurrentPin', newVal)
+      }
+    }
+  },
+
+  methods: {
+    createMap() {
+      if (!settings.mapboxToken) console.error('Please add a mapboxToken in config.json')
```
Comment (kenmickles) [inline template/components/Map.vue:218]: So polite!

**Pull request merged**